Gems/LmbrCentral/Code/Source/Shape/EditorTubeShapeComponentMode.cpp

```diff
@@ -268,6 +350,27 @@ namespace LmbrCentral
         ContainerChanged();
     }
 
+    void EditorTubeShapeComponentMode::ResetRadii()
```

Comment:
question: I take it the code in `PopulateActionsImpl` in this file will be removed when the feature flag gets removed and all the new code is turned on by default? (might be worth maybe moving this to a function to avoid the duplication in both places but it's not a huge deal).

Reply:
Correct. I will try to remove the duplication if it's not too complex.

Reply:
Removed duplicate code

Reply:
Awesome thanks! 🙂